Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,138 @@
+/*GLOBAL STLES*/
+
+html {
+	padding: 1%;
+	border-bottom: 15px solid #005C89;
+	border-top: 15px solid #005C89;
+}
+
+nav {
+	text-align: center;
+	font-family: Open Sans, sans-serif;
```

Comment:
To be safe, you'll want to put custom font names in single quotes since they're not globally recognized names: `'Open Sans', sans-serif;`